content/src/main/content/jcr_root/apps/acs-tools/components/tag-maker/clientlibs/js/app.js

```diff
@@ -68,6 +68,9 @@ angular.module('acs-tools-tag-maker-app', ['ngFileUpload', 'acsCoral', 'ACS.Tool
                 tagIds: []
             };
 
+            NotificationsService.add('info',
```

Comment:
@davidjgonzalez  I didn't notice that utility. Thanks for the suggestion. I'll change the code, test it and update the PR.